docs/cli/ilab-local-config.md

```diff
@@ -0,0 +1,184 @@
+# InstructLab CLI Local Storage Proposal
+
+## Why is this needed?
+
+For tools to be useful to the user, they generally need to work on and know about data pertinent to the user. For example, podman and docker track image blobs, kubectl tracks cluster credentials, etc.
+
+The InstructLab CLI manages a number of settings today such as the teacher model, taxonomy location, inference model, etc. using a local file known as `config.yaml`.
+
+While this method allows ilab to avoid additional complexity of managing a dedicated directory, it’s limited by its ability to scale beyond constraining the user to run ilab in a single directory.
+Since the new ilab CLI will need to not only manage datasets, downloaded models, and any intermediary files, it'll also have to make it simple for the user to orchestrate these things.
```

Comment:
What is "the new ilab CLI"?

Reply:
The one we're currently building for RHEL AI.

Reply:
How would you rephrase this?

Reply:
Well that depends on what you are trying to say - if the "new ilab CLI" is one that consumes the new libraries, I would put it in the context of that, since that is something that is ubiquitous both upstream and downstream.

Reply:
I like that more, I'll update it.